<launch>
      <node name="airsim_pose" pkg="airsim" type="pose.py" respawn="false" output="screen">
            <param name="frame_id" type="string" value="world" />
            <param name="pub_node" type="string" value="airsim/pose" />
            <param name="vehicle_name" type="string" value="airsimvehicle" />
      </node>
      <node name="airsim_pose_tf" pkg="airsim" type="airsim_pose_tf_broadcaster.py" respawn="false" output="screen">
            <param name="frame_id" type="string" value="world" />
            <param name="child_frame_id" type="string" value="base_link" />
            <param name="pose_node" type="string" value="airsim/pose" />
      </node>
      <node name="airsim_imu" pkg="airsim" type="imu.py" respawn="false" output="screen">
            <param name="frame_id" type="string" value="base_link" />
            <param name="pub_node" type="string" value="airsim/imu" />
            <param name="sensor_name" type="string" value="imu" />
            <param name="vehicle_name" type="string" value="airsimvehicle" />
      </node>
      <node name="airsim_lidar" pkg="airsim" type="lidar.py" respawn="false" output="screen">
            <param name="frame_id" type="string" value="base_laser" />
            <param name="pub_node" type="string" value="airsim/lidar" />
            <param name="sensor_name" type="string" value="lidar" />
            <param name="vehicle_name" type="string" value="airsimvehicle" />
      </node>
      <node pkg="tf" type="static_transform_publisher" name="base_to_laser_broadcaster" args="0 0 0 0 0 0 base_link base_laser 100" />
      <node pkg="rosbag" type="record" name="rosbag_record" args="record -o /home/airsim/airsim_lidar_benchmark -a"/>
</launch>